src/main/resources/application.conf

```diff
@@ -12,65 +12,29 @@ kamon {
 
   show-aspectj-missing-warning = on
 
-  metric {
-    filters {
-      # Define filter for your actor names here
-      # Example 1: akka-actor.includes = [ "default/user/**" ]
-      # Example 2:
-      # akka-actor {
-      # includes = [ "my-app/user/job-manager", "my-app/user/worker-*" ]
-      # excludes = [ "my-app/system/**", "my-app/user/worker-helper" ]
-      # }
-      # Example 3:
-      akka-actor.includes = [ "default/user/testActor1", "default/user/testActor2" ]
-      akka-router.includes = [ "**" ]
-      akka-dispatcher.includes = [ "**" ]
-      trace.includes =  [ "**" ]
-      trace-segment.includes =  [ "**" ]
-      histogram.includes =  [ "**" ]
-      min-max-counter.includes =  [ "**" ]
-      gauge.includes =  [ "**" ]
-      counters.includes =  [ "**" ]
-      # For web projects
-      http-server.includes = [ "**" ]
-
-      counters.excludes =  [ "" ]
-      trace-segment.excludes =  [ "" ]
-      histogram.excludes =  [ "" ]
-      min-max-counter.excludes =  [ "" ]
-      gauge.excludes =  [ "" ]
-      http-server.excludes =  [ "" ]
-    }
-  }
+  reporters = ["kamon.spm.SPMReporter"]
 
   spm {
-    token = "[place-token-here]"
+    token = "4a1366c2-9b4a-4f91-a0d2-c83b2e13e9a9"
     #Proxy support will be added in 0.6.8 version
     #proxy-server = "127.0.0.1"
     #proxy-port = 3128
     #proxy-user = "user"
     #proxy-password = "password"
+  }
+}
+
+kamon.util.filters {
+
+  "akka.tracked-actor" {
+    includes = [ "default/user/testActor1", "default/user/testActor2" ]
+  }
 
-    # Optional
-    subscriptions {
```

Comment:
What happens with all these subscriptions in Kamon 1.x?  They are simply not needed? Or they just don't perform any function in the context of Sematext's monitoring integration with Kamon?  Or they are not supported in 1.x any more?

Reply:
Subscriptions protocol is gone in 1.x - https://kamon.io/documentation/1.x/recipes/migrating-from-kamon-0.6/, since they remove akka dependency in 1.x